R/chromote.R

```diff
@@ -1,7 +1,9 @@
 #' Chromote class
 #'
+#' @description
 #' This class represents the browser as a whole.
 #'
+#' @details
```

Comment:
```suggestion
#' @description
```

The first sentence is not very useful indeed.